src/main/java/org/broadinstitute/consent/http/service/users/handler/DACUserRolesHandler.java

```diff
@@ -185,31 +187,14 @@ public void updateRoles(Map<String, DACUser> usersMap) throws UserRoleHandlerExc
         }
     }
 
-    private void updateAdminEmailPreference(List<UserRole> originalRoles, List<UserRole> updatedRoles, Integer dacUserId) throws UserRoleHandlerException {
```

Comment:
This method, and all attempts to manage email by role membership is no longer a valid approach. Instead, we manage the user's email settings directly from the UI.